bin/deliver_poste_messages

```diff
@@ -270,18 +271,21 @@ def main()
     until queue.empty?
       next unless delivery = queue.pop(true) rescue nil
 
+      sent_at = DateTime.now
+
       begin
         deliverer.send delivery
-      rescue Net::SMTPSyntaxError => e
-        puts "Unable to deliver #{delivery[:id]} to '#{delivery[:contact_email]}' because '#{e.message.to_s.strip}'"
-      rescue Net::SMTPFatalError => e
+      rescue Net::SMTPSyntaxError, Net::SMTPFatalError => e
         puts "Unable to deliver #{delivery[:id]} to '#{delivery[:contact_email]}' because '#{e.message.to_s.strip}'"
```

Comment:
Can you think of any case in which an SMTP syntax or fatal error should be reattempted? Assuming now, then I agree that makes sense.